content/en/blog/2023/kubecon-na.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at KubeCon NA 2023
+linkTitle: KubeCon NA '23
+date: 2023-09-11
+# prettier-ignore
+cSpell:ignore: Contribfest Pivotto Purvi Kanal Flamegraphs Sharone Zitzman Itiel Shwartz Komodor Ohly Shivanshu Shrivastava Kota Endo Masanori Itoh Danielson Benedikt Bongartz Mirabella Matej Coralogix Anusha Reddy Narapureddy Jaglowski observ Chronosphere Aronoff Broadbridge Hrabovcak Shishi
+author: '[Severin Neumann](https://github.com/svrnm)'
+---
+
+The OpenTelemetry project maintainers, members of the governance committee, and
+technical committee are excited to be at [KubeCon NA][] in Chicago from November
```

Comment:
> excited

Just a pet peeve, but I noticed that most blog announcements have at least one or two instances of "excited". This is understandable, but perhaps might get a bit tiring for the dedicated follower to read? "Woof, OTel folks are always excited!" :-)

Reply:
🤣 ... fair point, there are alternatives I guess